src/easydmp/dmpt/templates/widgets/formset.html

```diff
@@ -17,6 +17,7 @@
     </div>
     {% endfor %}
 </div>
+{% if not formset.max_num %}
```

Comment:
It turns out that max_num if explicitly unset is implicitly set to 1000, so we can't do it this way.

`/django/forms/formsets.py`: ` DEFAULT_MAX_NUM = 1000`